### PR TITLE
Backport fix for thrift dependency issue & fix for ENTESB-12350

### PR DESCRIPTION
**Pull request by phantomjinx**

### CODE REVIEW

File: install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go [hunk 1]
```diff
@@ -73,7 +73,8 @@ type ComponentsSpec struct {
 }
 
 type OauthConfiguration struct {
-	Tag string `json:"tag,omitempty"`
+	DisableSarCheck *bool  `json:"disable-sar-check,omitempty"`
```
Comment (lgarciaaco) [inline install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go]: Can we use camel here?

Comment (heiko-braun) [reply install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go]: You mean `DisableSarCheck`, rather than `disable-sar-check`?

Comment (heiko-braun) [reply install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go]: I think we should use the same property as in 1.7.x

Comment (lgarciaaco) [reply install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go]: Ok I meant dromedary case: `disableSarCheck`. All properties for CR use that case

Comment (lgarciaaco) [reply install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go]: Now I see in master it is also called `disable-sar-check` and I can't find that property in 1.7.x :thinking:

Comment (phantomjinx) [reply install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go]: Consistancy with master I think should be kept.

Comment (lgarciaaco) [reply install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go]: I will change master ...

Comment (phantomjinx) [reply install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go]: Done.

**Comment by phantomjinx** [comment]
No need to port to 1.9.x branch. Associated code already present.

**Pull request merged**